src/app/children/child-details/child-details.component.html

```diff
@@ -17,13 +17,9 @@
 
 <div class="row">
```

Comment:
Do those divs and their classes still have meaning? Let's simplify and remove anything that's not strictly necessary (including the commented out sections of the template).